src/rred_reports/redcap/main.py

```diff
@@ -23,14 +25,16 @@ class ExtractInput:
 class RedcapReader:
     """Reads two years of redcap data, processing the files (wide to long, and others) and filtering to non-empty rows"""
 
-    def __init__(self, school_list: Path):
-        """
-        Setup for reading from redcap
-
-        Args:
-            school_list: path to Excel dispatch list file
-        """
+    def __init__(self, school_list: Path, school_aliases: Optional[Path] = None):
         self._school_list = get_unique_schools(school_list)
+        self._school_aliases = Path("input/school_aliases/template.toml")
```

Comment:
Ah I think this is why the other tests are failing, we want this to default to `None`, so that if you don't set an override file then nothing happnens. The setup already has that you can call the `extract` function or CLI with the `school_aliases` to pass in a file
```suggestion
        self._school_aliases = None
```